include/circt/Firtool/Firtool.h

```diff
@@ -113,14 +113,19 @@ struct FirtoolOptions {
       llvm::cl::desc("Disable deduplication of structurally identical modules"),
       llvm::cl::init(false), llvm::cl::cat(category)};
 
-  llvm::cl::opt<bool> grandCentralInstantiateCompanionOnly{
-      "grand-central-instantiate-companion",
-      llvm::cl::desc("Run Grand Central in a mode where the companion module "
-                     "is instantiated and not bound in and the interface is "
-                     "dropped.  This is intended for situations where there is "
-                     "useful assertion logic inside the companion, but you "
-                     "don't care about the actual interface."),
-      llvm::cl::init(false), llvm::cl::Hidden, llvm::cl::cat(category)};
+  llvm::cl::opt<firrtl::CompanionMode> companionMode{
+      "grand-central-companion-mode",
+      llvm::cl::desc("Disable grand central and remove all collateral"),
```

Comment:
Description is not quite right. This changes the behavior of how the grand central _view_ companion is lowered.